language/tools/move-package/src/source_package/manifest_parser.rs

```diff
@@ -213,7 +213,10 @@ pub fn parse_addresses(tval: TV) -> Result<PM::AddressDeclarations> {
                         } else if addresses
                             .insert(
                                 ident,
-                                Some(parse_address_literal(entry_str).context("Invalid address")?),
+                                Some(
+                                    AccountAddress::from_hex_fuzzy(entry_str)
```

Comment:
I think these should not be fuzzy. We should force 0x for package addresses

Reply:
Didn't we add a test for that on the last PR?

Reply:
The original one `parse_address_literal` was fuzzy, so I'm unsure there.  I can switch it back.

Reply:
Let's make sure we add back the dropped in tests from #59, those should test this change here one way or another